README.md

```diff
@@ -1,71 +1,40 @@
 # React Truffle Box
 
-This box comes with everything you need to start using smart contracts from a react app. This is as barebones as it gets, so nothing stands in your way.
+This box comes with everything you need to start using Truffle to write, compile, test, and deploy smart contracts, and interact with them from a React app.
 
 ## Installation
 
-First ensure you are in a new and empty directory.
+First ensure you are in an empty directory.
 
-1. Run the `unbox` command via `npx` and skip to step 3. This will install all necessary dependencies. A Create-React-App is generated in the `client` directory.
-   ```js
-   npx truffle unbox react
-   ```
+Run the `unbox` command using 1 of 2 ways.
 
-2. Alternatively, you can install Truffle globally and run the `unbox` command.
-    ```javascript
-    npm install -g truffle
-    truffle unbox react
-    ```
+```sh
+# Install Truffle globally and run `truffle unbox`
+$ npm install -g truffle
+$ truffle unbox react
+```
 
-3. Run the development console.
-    ```javascript
-    truffle develop
-    ```
+```sh
+# Alternatively, run `truffle unbox` via npx
+$ npx truffle unbox react
+```
 
-4. Compile and migrate the smart contracts. Note inside the development console we don't preface commands with `truffle`.
-    ```javascript
-    compile
-    migrate
-    ```
+Start the react dev server.
 
-5. In the `client` directory, we run the React app. Smart contract changes must be manually recompiled and migrated.
-    ```javascript
-    // in another terminal (i.e. not in the truffle develop prompt)
-    cd client
-    npm run start
-    ```
+```sh
+$ cd client
```

Comment:
I think we need a `npm install in here` before we can do `npm run start`

Reply:
ahhhh gotcha I didn't unbox I cloned which makes sense why I didn't work for me. Sick UX though if that's the case!